lumenx/assetlist.json

```diff
@@ -1,9 +1,9 @@
 {
     "$schema":"../assetlist.schema.json",
-    "chain_name":"lumenx",
+    "chain_name":"lumen",
```

Comment:
Please don't change this chain_name (which must match the folder name).

Reply:
This would be the last blocker

Reply:
Wow!